core/task/channel/channel.go

```diff
@@ -25,33 +25,33 @@
 package channel
 
 import (
+	"errors"
 	"strconv"
 	"strings"
-	"errors"
 
 	"github.com/AliceO2Group/Control/common/logger"
 	"github.com/sirupsen/logrus"
 )
 
 var log = logger.New(logrus.StandardLogger(), "channel")
 
-type channel struct {
-	Name        string                  `yaml:"name"`
-	Type        ChannelType             `yaml:"type"`
-	SndBufSize  int                     `yaml:"sndBufSize"`
-	RcvBufSize  int                     `yaml:"rcvBufSize"`
-	RateLogging int                     `yaml:"rateLogging"`
-	Transport   TransportType           `yaml:"transport"`  //default: default
+type Channel struct {
+	Name        string        `yaml:"name"`
+	Type        ChannelType   `yaml:"type"`
+	SndBufSize  int           `yaml:"sndBufSize"`
+	RcvBufSize  int           `yaml:"rcvBufSize"`
+	RateLogging int           `yaml:"rateLogging"`
+	Transport   TransportType `yaml:"transport"` //default: default
```

Comment:
No need to change back these files, but I suggest you disable formatting on save in your IDE. I like nicely formatted code but commits should generally only touch what they need to touch, otherwise it becomes difficult later on to understand which part of the commit is a functional change as opposed to formatting.